lib/ansible/modules/apt.py

```diff
@@ -372,7 +372,7 @@
 from ansible.module_utils.common.locale import get_best_parsable_locale
 from ansible.module_utils.common.respawn import has_respawned, probe_interpreters_for_module, respawn_module
 from ansible.module_utils.common.text.converters import to_native, to_text
-from ansible.module_utils.six import PY3, string_types
+from ansible.module_utils.six import string_types
```

Comment:
`six.string_types` can be unconditionally replaced with `str` everywhere per https://six.rtfd.io/#six.string_types
```suggestion
```